client/app/routes/routes.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Route } from 'react-router';
+import Layout from '../layout/Layout';
+import TestReactRouter from '../components/TestReactRouter';
+import TestReactRouterRedirect from '../components/TestReactRouterRedirect';
+import CommentScreen from '../components/CommentScreen';
+
+export default (
+  <Route component={Layout}>
+    <Route
+      path="/"
+      component={CommentScreen}
+    />
+    <Route
+      path="/react-router"
+      component={TestReactRouter}
+    />
+    <Route
+      path="/react-router/redirect"
+      component={TestReactRouterRedirect}
+      onEnter={TestReactRouterRedirect.checkAuth}
```

Comment:
This is where we're check server rendering doing the right thing for a redirect.